### PR TITLE
OSDOCS-10888: 4.15.18 RN

### PR DESCRIPTION
**Pull request by obrown1205**
Version(s):
4.15

Issue:
[OSDOCS-10888](https://issues.redhat.com/browse/OSDOCS-10888)

Link to docs preview:
https://77624--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-15-release-notes.html#ocp-4-15-18

QE review:
N/A

Additional information:
Errata URLs will return 404 until they are shipped live

### CODE REVIEW

File: release_notes/ocp-4-15-release-notes.adoc [hunk 1]
```diff
@@ -2730,8 +2730,65 @@ This section will continue to be updated over time to provide notes on enhanceme
...
+==== Enhancements
+
+The following enhancements are included in this z-stream release:
+
+[id="ocp-4-15-18-unservable-future-versions-status-condition"]
+===== Introducing the UnservableFutureVersions status condition for SHA1 routes
+
+* {product-title} {product-version} does not support SHA1 certificates on routes. Consequently, upgrades from 4.15 to {product-version} will cause routes with SHA1 certificates to be rejected. 
++
+This update introduces a new `UnservableInFutureVersions` status condition to routes that contain a SHA1 certificate. Additionally, it adds an `admin-gate` to block upgrades if this new status is present on any route. As a result, if cluster administrators have routes that use SHA1 certificates in {product-title} 4.15, they must either upgrade these certificates to a support algorithm or provide an `admin-ack` for the created `admin-gate`. This `admin-ack` allows administrators to proceed with the upgrade without resolving the SHA1 certificate issues, even though the routes will be rejected. (link:https://issues.redhat.com/browse/OCPBUGS-28928[*OCPBUGS-28928*]).
```
Comment (mburke5678) [inline release_notes/ocp-4-15-release-notes.adoc:2759]:
```suggestion
This update adds a new `UnservableInFutureVersions` status condition to routes that contain a SHA1 certificate. Additionally, it adds an `admin-gate` to block upgrades if this new status is present on any route. As a result, if cluster administrators have routes that use SHA1 certificates in {product-title} 4.15, they must either upgrade these certificates to a support algorithm or provide an `admin-ack` for the created `admin-gate`. This `admin-ack` allows administrators to proceed with the upgrade without resolving the SHA1 certificate issues, even though the routes will be rejected. (link:https://issues.redhat.com/browse/OCPBUGS-28928[*OCPBUGS-28928*]).
```

File: release_notes/ocp-4-15-release-notes.adoc [hunk 2]
```diff
@@ -2730,8 +2730,65 @@ This section will continue to be updated over time to provide notes on enhanceme
...
+$ oc adm release info 4.15.18 --pullspecs
+----
+
+[id="ocp-4-15-18-enhancements"]
+==== Enhancements
+
+The following enhancements are included in this z-stream release:
+
+[id="ocp-4-15-18-unservable-future-versions-status-condition"]
+===== Introducing the UnservableFutureVersions status condition for SHA1 routes
```
Comment (mburke5678) [inline release_notes/ocp-4-15-release-notes.adoc:2755]:
```suggestion
===== UnservableFutureVersions status condition for SHA1 routes
```

File: release_notes/ocp-4-15-release-notes.adoc [hunk 3]
```diff
@@ -2730,8 +2730,65 @@ This section will continue to be updated over time to provide notes on enhanceme
...
+
+* {product-title} {product-version} does not support SHA1 certificates on routes. Consequently, upgrades from 4.15 to {product-version} will cause routes with SHA1 certificates to be rejected. 
++
+This update introduces a new `UnservableInFutureVersions` status condition to routes that contain a SHA1 certificate. Additionally, it adds an `admin-gate` to block upgrades if this new status is present on any route. As a result, if cluster administrators have routes that use SHA1 certificates in {product-title} 4.15, they must either upgrade these certificates to a support algorithm or provide an `admin-ack` for the created `admin-gate`. This `admin-ack` allows administrators to proceed with the upgrade without resolving the SHA1 certificate issues, even though the routes will be rejected. (link:https://issues.redhat.com/browse/OCPBUGS-28928[*OCPBUGS-28928*]).
+
+[id="ocp-4-15-18-password-colon-character"]
+===== Allowing pull secret passwords to contain a colon character
+
+* This release introduces the ability to include a colon character in your password for the Openshift Assisted Installer. (link:https://issues.redhat.com/browse/OCPBUGS-34641[*OCPBUGS-34641*])
```
Comment (mburke5678) [inline release_notes/ocp-4-15-release-notes.adoc:2764]:
```suggestion
* This release adds the ability to include a colon character in your password for the Openshift Assisted Installer. (link:https://issues.redhat.com/browse/OCPBUGS-34641[*OCPBUGS-34641*])
```

File: release_notes/ocp-4-15-release-notes.adoc [hunk 4]
```diff
@@ -2730,8 +2730,65 @@ This section will continue to be updated over time to provide notes on enhanceme
...
+
+[id="ocp-4-15-18-defragment-etcd"]
+===== Introducing an etcd defragmentation controller for Hypershift
+
+* This release introduces an etcd degragmentation controller for hosted clusters on Hypershift. (link:https://issues.redhat.com/browse/OCPBUGS-35002[*OCPBUGS-35002*])
```
Comment (mburke5678) [inline release_notes/ocp-4-15-release-notes.adoc:2769]:
```suggestion
* This release add an etcd degragmentation controller for hosted clusters on Hypershift. (link:https://issues.redhat.com/browse/OCPBUGS-35002[*OCPBUGS-35002*])
```

**Comment by mburke5678** [comment]
@obrown1205 A few nits. Feel free to ignore.

**Comment by bscott-rh** [comment]
Merge review LGTM.

**Pull request merged**